usr/src/test/zfs-tests/tests/functional/grow_replicas/Makefile

```diff
@@ -18,11 +18,9 @@ include $(SRC)/Makefile.master
 ROOTOPTPKG = $(ROOT)/opt/zfs-tests
 TESTDIR = $(ROOTOPTPKG)/tests/functional/grow_replicas
 
-PROGS = cleanup \
-	grow_replicas_001_pos \
-	setup
+PROGS=	grow_replicas_001_pos
```

Comment:
Please leave spaces around equal sign to match the convention used elsewhere under usr/test.